### DISCUSSION TITLE
Licence pve remove sub warning

### DISCUSSION THREAD

**Discussion by tia9os** [lae/ansible-role-proxmox#159] [Q&A]
Hello dev Team,

please consider, that using the parameter "pve_remove_subscription_warning" might be illegal,
due to the the GPLv3 licence proxmox is released under(every code modification needs to be made publicly available).

Please check if this complies with your code on https://www.gnu.org/licenses/why-affero-gpl.

**Comment by lae** [comment]
I can't see how the parameter itself is illegal. But the patches themselves could be marked under GPL?

**Comment by jmariondev** [comment]
I agree, it seems to me like we should perhaps explicitly mark the patch as AGPL (since it contains AGPL code from upstream in the form of context lines).

Otherwise, the patched file is just JavaScript which is sent to the client anyway. The modified source and patch are already available to anyone that wants it–"free" in every sense–from the Proxmox interface and here in this repository.

I am, however, not a lawyer. I am willing to defer to others that are more knowledgeable. Ultimately, this is @lae's call.
